src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -306,16 +306,7 @@ func isPrivate(ip net.IP) (bool, error) {
 
 const helpBootstrap = `Initialize the configuration to bootstrap a cluster.
 
---id is mandatory. bootstrap will expect the machine it's running on
-to have only one private non-loopback IP address associated to it,
-and use it in the configuration as the node's address.
-
-If it has multiple IPs, --self must be specified.
-In that case, the given IP will be used without checking whether it's
-among the machine's addresses or not.
-
-The elements in --ips must be separated by a comma, no spaces.
-
-If omitted, the node will be configured as a root node, that other
-ones can join later.
+This command expects the machine it's running on to have only one private
+non-loopback IP address associated to it, and use it in the configuration as
+the node's address. If it has multiple IPs, --self must be specified.
```

Comment:
I think this text is now a bit too brief, and the before and after versions could do a bit more explaining what this command does (even if it's not much). What do you think about:

```
Initialize the configuration to bootstrap a cluster.

This command generates a redpanda.yaml configuration file to bootstrap a
cluster. If you are modifying the configuration file further, it is recommended
to first bootstrap and then modify. If the file already exists, this command
will set fields as requested by flags, and this may undo some of your earlier
edits.

The --ips flag specifies seed servers that this broker will use to form a
cluster.

By default, redpanda expects your machine to have one private IP address, and
redpanda will listen on it. If your machine has multiple private IP addresses,
you must use the --self flag to specify which ip redpanda should listen on.
```
Also, this be moved inline in the command? We don't separate the help text usually -- I've only done that when the help text gets extremely long.

Reply:
Done